### DISCUSSION TITLE
VideoReDo feature

### DISCUSSION THREAD

**Discussion by mstick99** [lart2150/kmttg#29] [General]
As part of the next release, maybe consider depreciating the VideoReDo feature since it is no longer available.

**Comment by renegaudet** [comment]
Glad you said deprecate instead of remove.  I continue to use both kmttg & videoredo together on a daily basis.  I would hate to see it removed, but I understand Videoredo is no longer available to new users.

**Comment by thess** [comment]
I don't think that VideoReDo support should be deprecated. My understanding of 'deprecated' is that the feature is obsolete and will be phased out. This is not the case here. VideoReDo is a legacy feature for a product no longer available. However, there a lot of product owners out there who would not like to see the feature removed. If you have a valid license, you can still activate new installs manually at: https://videoredo.online

See discussion: https://www.reddit.com/r/videoredo/

**Comment by lart2150** [comment]
In some regards all of kmttg is deprecated as I don't **plan** on adding new features, I'm mainly keeping the lights on supporting newer versions of java and keeping certs up to date. With that said I have no plans on removing any features until there's a good reason to, Like Thess has said you can still activate existing licenses thanks to the efforts of Dan203.
